### DISCUSSION TITLE
Testimonial Section

### DISCUSSION THREAD

**Discussion by dhruvabhat24** [EddieHubCommunity/LinkFree#10048] [Q&A] [answered]
My question is that how could i get a testmonial on my profile. 
Evn i am not able to add the testimonials. 
Even when someone views my profile they are also not able to provide a testimonial 
Can you help me about explaining the procedure

**Comment by ArvindParekh** [comment, marked as answer]
Hey @dhruvabhat24. Check out the procedure in the docs [here](https://www.biodrop.io/docs/how-to-guides/testimonials-json). It should walk you through getting testimonials on your profile. I hope that helps.

**Comment by dhruvabhat24** [comment]
Thank uoy @ArvindParekh

**Comment by dhruvabhat24** [comment]
![Screenshot 2023-12-29 185655](https://github.com/EddieHubCommunity/BioDrop/assets/122305929/af6dc072-fab1-4239-8b12-f12acff87ebd)
As per the documentation in need to get a botton if i am providing testimonial for others but i am not getting it .

**Reply by ArvindParekh** [reply]
You're right. I did take a look at the docs after I replied to you, and the button suggested in the docs that says `Add testimonial` on a user's profile has now been removed. We're currently in the process of updating the docs. Meanwhile, if you want to add a testimonial, you can always go the the [Issues Tab](https://github.com/eddiehubcommunity/biodrop/issues) of the BioDrop repo, click on `New Issue` and select the type 'Testimonial'.

**Reply by dhruvabhat24** [reply]
Now if anyone wants to give a testimonial to me then even they need to follow same procedure right ?

**Reply by ArvindParekh** [reply]
That's correct.

**Reply by dhruvabhat24** [reply]
Okay Thankyou

**Reply by eddiejaoude** [reply]
Sorry for the confusion! BTW Great collaboration 💪 

Our docs need updating, the button got removed, but testimonials can still be added via the issue template here 
https://github.com/EddieHubCommunity/BioDrop/issues/new?assignees=&labels=testimonial&projects=&template=testimonial.yml&title=New+Testimonial

**Reply by ArvindParekh** [reply]
Eddie, I'm just curious. Why was the button removed in the first place? Did it interfere with something else we were planning?

**Reply by dhruvabhat24** [reply]
Thankyou @eddiejaoude

**Reply by ArvindParekh** [reply]
> Eddie, I'm just curious. Why was the button removed in the first place? Did it interfere with something else we were planning?

Hi @eddiejaoude. Looks like this got lost in the conversation. Still curious about this :)

**Comment by dhruvabhat24** [comment]
I have recieved one testimonial even it is closed. 
But in BioDrop I am not getting anything

![Screenshot 2023-12-31 145319](https://github.com/EddieHubCommunity/BioDrop/assets/122305929/039abc60-38b6-4e40-bbb9-9f9964d272f1)
![Screenshot 2023-12-31 145332](https://github.com/EddieHubCommunity/BioDrop/assets/122305929/bf3df12d-04ba-4efb-8e2d-1fd2919860a2)

**Reply by eddiejaoude** [reply]
oh that is strange, please raise an issue (bug) with these screenshots

**Reply by dhruvabhat24** [reply]
Okay

**Comment by dhruvabhat24** [comment]
@eddiejaoude  No need of raising an issues again Now I am able to add it 
But i think it will take some to fetch the details.
Sgould i need to raise it as a issue ...?

**Reply by eddiejaoude** [reply]
We are having multiple conversations in different places, yes please raise the issue and the conversation can continue there to avoid confusion

**Reply by dhruvabhat24** [reply]
I have raised issues but it is closed as already solved. And duplicate
